plugin/trino-hive/src/main/java/io/trino/plugin/hive/avro/TrinoAvroSerDe.java

```diff
@@ -146,7 +146,7 @@ private static Schema getSchemaFromFS(String schemaFSUrl, Configuration conf)
      */
     public static Schema getSchemaFor(InputStream stream)
     {
-        Schema.Parser parser = new Schema.Parser();
+        Schema.Parser parser = new Schema.Parser().setValidateDefaults(false);
```

Comment:
Add a link to https://issues.apache.org/jira/browse/HIVE-24797 and link to similar code in apache/hive master branch

BTW -- the "Copied from" above just became outdated, unintentionally. This is why it's better not to have it in the code, but rather have it as a commit message.

Reply:
I've replaced the "copied from" comments with to-do comments saying to remove the methods after the issue is fixed in Hive, one comment noting the change and linking the fix in Hive, and one comment saying that the methods were _originally_ copied (to avoid making it sound like they remain unmodified).

Though maybe a better temporary solution would also say "do not modify these methods," because we don't actually want to diverge from Hive at all here.